src/frontend/packages/lib_classroom/src/components/ClassroomWidgetProvider/widgets/ToolsAndApplications/index.tsx

```diff
@@ -0,0 +1,179 @@
+    id: 'component.ToolsAndApplications.enableChat',
+  },
+  enableSharedNotes: {
+    defaultMessage: 'Enable shared notes',
+    description: 'Label for the switch that enables or disables shared notes',
+    id: 'component.ToolsAndApplications.enableSharedNotes',
+  },
+  enableBeakoutRooms: {
+    defaultMessage: 'Enable waiting room',
+    description: 'Label for the switch that enables or disables waiting room',
+    id: 'component.ToolsAndApplications.enableBeakoutRooms',
+  },
+  enablePresentationSupports: {
+    defaultMessage: 'Enable presentation supports sharing',
+    description:
+      'Label for the switch that enables or disables presentation support sharing',
+    id: 'component.ToolsAndApplications.enablePresentationSupports',
+  },
+  enableRecordings: {
+    defaultMessage: 'Enable recordings',
+    description: 'Label for the switch that enables or disables recordings',
+    id: 'component.ToolsAndApplications.enableRecordings',
+  },
+  updateClassroomSuccess: {
+    defaultMessage: 'Classroom updated.',
+    description: 'Message when classroom update is successful.',
+    id: 'component.ToolsAndApplications.updateClassroomSuccess',
+  },
+  updateClassroomFail: {
+    defaultMessage: 'Classroom not updated!',
+    description: 'Message when classroom update failed.',
+    id: 'component.ToolsAndApplications.updateClassroomFail',
+  },
+});
+
+export const ToolsAndApplications = () => {
+  const classroom = useCurrentClassroom();
+  const intl = useIntl();
+  const updateClassroomMutation = useUpdateClassroom(classroom.id, {
+    onSuccess: () => {
+      toast.success(intl.formatMessage(messages.updateClassroomSuccess));
+    },
+    onError: () => {
+      toast.error(intl.formatMessage(messages.updateClassroomFail));
+      setUpdatedClassroomState(classroom);
+    },
+  });
+  const [updatedClassroomState, setUpdatedClassroomState] = useState<
+    Partial<Classroom>
+  >({});
+
+  const handleChange = (updatedClassroomAttribute: Partial<Classroom>) => {
+    const updatedClassroom = {
+      ...updatedClassroomState,
+      ...updatedClassroomAttribute,
+    };
+    setUpdatedClassroomState(updatedClassroom);
+    if (JSON.stringify(updatedClassroom) !== '{}') {
+      updateClassroomMutation.mutate(updatedClassroom);
+    }
+  };
+
+  useEffect(() => {
+    setUpdatedClassroomState({});
+  }, [classroom]);
+
+  return (
+    <FoldableItem
+      infoText={intl.formatMessage(messages.info)}
+      initialOpenValue
+      title={intl.formatMessage(messages.title)}
+    >
+      <Box
+        align="left"
+        background="blue-message"
+        fill
+        round="xsmall"
+        margin={{ bottom: 'small' }}
+      >
+        <ToggleInput
+          checked={classroom.enable_chat}
+          label={intl.formatMessage(messages.enableChat)}
+          onChange={() => handleChange({ enable_chat: !classroom.enable_chat })}
+        />
+      </Box>
```

Comment:
WDYT about extracting this to a component (which could stay in this file) for avoiding repetition ?